picard/ui/mainwindow.py

```diff
@@ -652,11 +670,17 @@ def create_menus(self):
     def update_toolbar_style(self):
         if config.setting["toolbar_show_labels"]:
             self.toolbar.setToolButtonStyle(QtCore.Qt.ToolButtonTextUnderIcon)
+            self.player_toolbar.setToolButtonStyle(QtCore.Qt.ToolButtonTextUnderIcon)
+            self.volume_label.show()
+            
```

Comment:
This empty line makes Codacy unhappy :)

Reply:
How to send new fixed version?

Reply:
@phw

Reply:
You can do any changes to your local branch, commit them and then push them back to this branch on Github. This will update the pull request automatically and trigger the test builds and Codacy check again.

Reply:
@phw 
I have fixed the problem. Can we merge it now?